README.md

```diff
@@ -0,0 +1,126 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ ] Please replace `# Project Name` with `# Math Magician` to make your readme file more descriptive :blush: